Dockerfile

```diff
@@ -71,4 +71,8 @@ COPY --from=builder /go/src/github.com/scality/mountpoint-s3-csi-driver/bin/scal
 # TODO: This won't be necessary with containerization.
 COPY --from=builder /go/src/github.com/scality/mountpoint-s3-csi-driver/bin/install-mp /bin/install-mp
 
+# Copy license files and attribution notices
+COPY --from=builder /go/src/github.com/scality/mountpoint-s3-csi-driver/LICENSES /LICENSES
+COPY --from=builder /go/src/github.com/scality/mountpoint-s3-csi-driver/NOTICE /NOTICE
```

Comment:
those are the less likely to change compared to source code, right ? So they should go above the `COPY` of source code so the layers won't need to change anytime source code changes and they can stay cached longer.

Reply:
Sound and cool observation, I changed it.
https://github.com/scality/mountpoint-s3-csi-driver/pull/71/commits/ab2778f3823b5158154f0817fe8c0182ad5a70ab